src/assets/components/ViewArticle.jsx

```diff
@@ -1,26 +1,56 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../../utils";
+import { getArticleById, getCommentByArticleId } from "../../utils";
 
 export default function ViewArticle() {
     const {article_id} = useParams()
     const [article, setArticle] = useState({})
+    const [comments, setComments] = useState([])
```

Comment:
Please create a state for loading and display a message to the user while the comments are loading.